PR (lda) (#107)
MrLongbottom/P9-Rapport: sections/method/PageRank.tex

where $p$ and $q$ are distributions and $m$ is the pointwise mean between $p$ and $q$. 
$D$ is the Kullback-Leibler divergence function.

The goal of using \gls{pr} is to get the most important documents within a specific topic and use these to yield important information during search.
  

> \gls{pr} does not take the query into account, which is the reason it is not used stand-alone. 

Reviewer: ```suggestion
\gls{pr} does not take the query into account, which is the reason it is only used in combination with other \gls{ir} methods. 
```

> \gls{pr} does not take the query into account, which is the reason it is only used in combination with other \gls{ir} methods. 

Reviewer 2: ```suggestion
\gls{pr} does not take the query into account, which is the reason why it is only used in combination with other \gls{ir} methods. 
```

> The \gls{pr} is to a certain degree connected with \gls{lda} since it uses the topic distributions created by the \gls{lda} to rank documents. 

Reviewer 2: ```suggestion
The \gls{pr} is to a certain degree connected with \gls{lda} since it uses the topic distributions, created by the \gls{lda}, to rank documents. 
```
